Add step to check if `pyenv` is usable by the user (#4449)
Scille/parsec-cloud: docs/development/quickstart.md

    4. [`python v3.9`](https://www.python.org/)

       To install the correct python version, we use `pyenv` instead of relaying on a system package:

       1. Install `pyenv`:

          ```shell
          # Install Pyenv
          curl --proto '=https' --tlsv1.2 -sSL https://pyenv.run | bash
          ```

          > For more installation methods for `pyenv`, see <https://github.com/pyenv/pyenv#installation>

       2. Check your `pyenv` installation:

>           If the following command work, you can skip this step
> 
>           ```shell
>           pyenv --version
>           ```

Reviewer: ```suggestion
          ```shell
          pyenv --version
          ```

          If the previous command worked, you can skip directly to step 3.
```

>           If this command failed with `command not found` (or alike) this mean your OS can't find `pyenv` inside our `PATH` variable.

Reviewer: ```suggestion
          If this command failed with `command not found` (or alike) this means your OS can't find `pyenv` inside your `PATH` variable.
```

>           To correct that you need to update your `PATH` variable, if you've installed `pyenv` via the step above you mostly need to add `$HOME/.pyenv/bin` to that variable.

Reviewer: ```suggestion
          To correct that you need to update your `PATH` variable. If you've installed `pyenv` via the step above you need to add `$HOME/.pyenv/bin` to that variable.
```

          > `$HOME` is you home path.

       3. Install the correct python version using `pyenv`

          ```shell

>           # Compile Python

Reviewer: ```suggestion
          # Install a specific Python version
```
